index.html

```diff
@@ -17,9 +17,10 @@
         <a href="https://stackoverflow.co/"> About </a>
         <p> Products</p>
         <a href="https://stackoverflow.co/teams/"> For teams </a>
-        <form class="relative">
-            <input placeholder=" Search..." type="text" name="search">
-            <svg class="search-icon absolute" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
+        <form class="relative text-input-with-icon">
```

Comment:
We have a bit of a problem. The `text-input-with-icon` is both on the `form` and on the `input` (line 21). It should only be on one of them. 

I think the best thing is to emove this class from here and leave it just on the input (line 21).

Then in CSS, change from  `.text-input-with-icon svg {` to `.text-input-with-icon + svg {` which will select all SVGs that follow directly a text-input with icon. I think it makes most sense like this.